README.md

```diff
@@ -65,4 +65,21 @@ For Snapshot and CDC modes, the following privileges are required:
 
 ## Destination
 
-(Planned)
+The mysql destination takes a `record.Record` and parses it into a valid SQL query. Each record is individually parsed and upserted, and write batching is planned to be implemented.
+
+### Upsert Behavior
+
+If the target table contains a column with a unique constraint (this includes PRIMARY KEY and UNIQUE indexes), records will be upserted; otherwise, they will be appended. Support for updating tables without unique constraints is tracked [here](https://github.com/conduitio-labs/conduit-connector-mysql/issues/66).
+
+### Multicollection mode
+
+(Planned to do). You can upvote [the following issue](https://github.com/conduitio-labs/conduit-connector-mysql/issues/13) to add more interest on getting this feature implemented sooner.
+
+### Configuration Options
+
+| name    | description                                                                                                                                                                           | required | default                                      |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | -------------------------------------------- |
+| `url`   | Connection string for the mysql database.                                                                                                                                          | true     |                                              |
+| `table` | The target table to write the record to | true    |  |
+| `key`   | Key represents the column name to use to delete records.                                                                                                 | false    |                                              |
```

Comment:
What's the default value and how does the connector behave when a key is not provided?

Reply:
Well, it was actually required, good catch. I wouldn't make it optional now, as in the near future the connector should be able to use multicollection to write to many tables.

The ideal would be to not even need to specify the key, and let the connector handle that at runtime.